src/azul/indexer/index_queue_service.py

```diff
@@ -326,12 +326,12 @@ def for_entity(cls,
                    attempts=0)
 
     @classmethod
-    def from_json(cls, json: JSON, attempts: int) -> Self:
-        return cls(entity=CataloguedEntityReference(catalog=json_str(json['catalog']),
-                                                    entity_type=json_str(json['entity_type']),
-                                                    entity_id=json_str(json['entity_id'])),
-                   num_contributions=json_int(json['num_contributions']),
-                   attempts=attempts)
+    def from_message(cls, msg: SQSFifoMessage) -> Self:
```

Comment:
Can we leave `from_json` in place for symmetry with `to_json` and make `from_message` call it?